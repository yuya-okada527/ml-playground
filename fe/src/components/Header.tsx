import {
  AppBar,
  createStyles,
  makeStyles,
  Theme,
  Typography,
} from "@material-ui/core";
import React from "react";

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    title: {
      margin: theme.spacing(2),
    },
  })
);

/**
 * ヘッダーを描画する
 */
const Header: React.FC = () => {
  const classes = useStyles();
  return (
    <header>
      <AppBar>
        <Typography variant="h4" component="h1" className={classes.title}>
          ML Playground
        </Typography>
      </AppBar>
    </header>
  );
};

export default Header;
