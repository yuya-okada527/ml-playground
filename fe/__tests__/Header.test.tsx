import React from "react";
import { shallow } from "enzyme";
import Header from "../src/components/Header";

describe("Header", () => {
  it("Snapshot Test", () => {
    const header = shallow(<Header />);
    expect(header).toMatchSnapshot();
  });
});
