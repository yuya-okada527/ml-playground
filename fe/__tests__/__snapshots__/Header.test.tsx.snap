// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot Test 1`] = `
<header>
  <WithStyles(ForwardRef(AppBar))>
    <WithStyles(ForwardRef(Typography))
      className="makeStyles-title-1"
      component="h1"
      variant="h4"
    >
      ML Playground
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(AppBar))>
</header>
`;
